catalogs/yale.yaml

```diff
@@ -6,6 +6,14 @@ sources:
   babylonian:
     description: Yale babylonian
     driver: sequential_csv
+    metadata:
+      data_path: yale/babylonian
+      post_harvest: "remove_babylon_non_relevant"
+      config: yale_babylon_config
```

Comment:
just because it doesn't quite fit the pattern of the other changes exactly, i'm wondering whether `_babylon_` in the two lines above is supposed to be `_babylonian_`?  though i also see it seems moved unchanged from below?